### ISSUE TITLE
Windows support?

### ISSUE THREAD

**Issue by nartamonov** [roomkey/lein-v#12]
It seems plugin does not work on Windows, because of dependency on `bash`:

```
java.io.IOException: Cannot run program "/bin/bash": CreateProcess error=2, ═х єфрхЄё  эрщЄш єърчрээ√щ Їрщы
 at java.lang.ProcessBuilder.start (:-1)
    java.lang.Runtime.exec (:-1)
    clojure.java.shell$sh.invokeStatic (shell.clj:113)
    clojure.java.shell$sh.doInvoke (shell.clj:79)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invokeStatic (core.clj:646)
    clojure.core$apply.invoke (core.clj:641)
    leiningen.v.git$eval655$git_command__656.invoke (git.clj:15)
    leiningen.v.git$git_describe.invokeStatic (git.clj:28)
    leiningen.v.git$git_describe.invoke (git.clj:27)
    leiningen.v.git$version.invokeStatic (git.clj:44)
    leiningen.v.git$version.doInvoke (git.clj:37)
...
```

Why can't we run git directly, [like it does](https://github.com/sbt/sbt-git/blob/master/src/main/scala/com/typesafe/sbt/git/ConsoleGitRunner.scala#L16) similar project `sbt-git` for Scala language?

**Comment by cch1** [comment]
TL;DR: my skills at running external executables from the JVM is not strong.

IIRC, running git outside bash means that the environment variables that tune its behavior are not present.  I would happily accept a pull request that works for both Windows and *nix shells. 

BTW, my long term goal is to query the git repo directly using the [JGit](https://eclipse.org/jgit/) library.  Unfortunately, the last time I started making the change the JGit library was missing some critical functionality.  That problem may now be resolved...

**Comment by timgilbert** [comment]
About the environment variables, `(shell/sh)` is implemented via `Runtime.exec()`, whose default behavior is to [have the subprocess inherit all of the parent's environment variables](https://docs.oracle.com/javase/8/docs/api/java/lang/Runtime.html#exec-java.lang.String:A-java.lang.String:A-java.io.File-), so the bash process isn't necessary for that purpose.

**Comment by timgilbert** [comment]
I think this should work now that #23 has been merged.
